packages/next/src/cli/next-dev.ts

```diff
@@ -202,12 +202,15 @@ const nextDev = async (
     traceUploadUrl = options.experimentalUploadTrace
   }
 
+  const distDir = path.join(dir, config.distDir ?? '.next')
```

Comment:
aside: Shouldn't this have gotten the default value closer to where we initialize `config`?

Reply:
Um I didn't add it but just moved above the original resolving!

https://github.com/vercel/next.js/pull/67157/files#diff-ddadbe5e107283e1bda7713050b8dcc19065dc804ab082e55e148efac0ef0221L219